services/TransactionService/composer.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "islandora/transaction-service",
+    "description": "RESTful service providing transactions in Fedora 4",
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "/Users/dpino/Desktop/Development/ISLANDORAWORK/CLAW_MICRO/chullo"
```

Comment:
Should this be the main github repo and just make this PR rely on the Chullo PR?